sodetlib/operations/bias_steps.py

```diff
@@ -960,7 +960,7 @@ def plot_Rfrac(bsa, text_loc=(0.6, 0.8)):
     chmap = sdl.map_band_chans(bsa.bands, bsa.channels,
                                iva.bands, iva.channels)
 
-    Rfracs = bsa.R0 / iva.R_n_IV[chmap]
+    Rfracs = bsa.R0 / iva.R_n[chmap]
```

Comment:
This change was already made in master, so heads up that you will probably need to manually merge some files when committing.

Reply:
will do